src/layout/page-layout.scss

```diff
@@ -0,0 +1,384 @@
+    --Layout-column-gap: #{$spacer-3}; // 16px
+
+    @include breakpoint(lg) {
+      --Layout-column-gap: #{$spacer-4}; // 24px
+    }
+  }
+
+  &.PageLayout--columnGap-condensed {
+    --Layout-column-gap: #{$spacer-3}; // 16px
+  }
+
+  &.PageLayout--columnGap-none {
+    // the `px` unit is mandatory for `calc()` execution. See https://stackoverflow.com/a/32518348
+    // stylelint-disable-next-line length-zero-no-unit
+    --Layout-column-gap: 0px;
+  }
+
+  // row gap
+
+  &.PageLayout--rowGap-normal {
+    --Layout-row-gap: #{$spacer-3}; // 16px
+
+    @include breakpoint(lg) {
+      --Layout-row-gap: #{$spacer-4}; // 24px
+    }
+  }
+
+  &.PageLayout--rowGap-none {
+    // the `px` unit is mandatory for `calc()` execution. See https://stackoverflow.com/a/32518348
+    // stylelint-disable length-zero-no-unit
+    --Layout-row-gap: 0px;
+  }
+
+  &.PageLayout--rowGap-condensed {
+    --Layout-row-gap: #{$spacer-3}; // 16px
+  }
+
+  // regions
+
+  .PageLayout-header,
+  .PageLayout-content,
+  .PageLayout-pane,
+  .PageLayout-footer {
+    // stylelint-disable-next-line primer/spacing
+    padding: var(--Layout-inner-spacing-min);
+  }
+
+  .PageLayout-content {
+    // stylelint-disable primer/spacing
+    padding-right: var(--Layout-inner-spacing-max);
+    padding-left: var(--Layout-inner-spacing-max);
+    // stylelint-enable primer/spacing
+    grid-area: content;
+  }
+
+  .PageLayout-pane {
+    grid-area: pane;
+  }
+
+  // multi-column desktop-friendly layout
+
+  @include breakpoint($Layout-responsive-variant-max-breakpoint) {
+
+    // Set a `content` region width, to work with loading states when
+    // `pane` is not yet loaded. See https://github.com/primer/css/pull/1818
+    $Layout-content-full-width: minmax(0, calc(100% - var(--Layout-pane-width) - var(--Layout-column-gap)));
+
+    &.PageLayout--panePos-start {
+      --Layout-template-columns: var(--Layout-pane-width) #{$Layout-content-full-width};
+      --Layout-template-areas: 'pane content';
+    }
+
+    &.PageLayout--panePos-end {
+      --Layout-template-columns: #{$Layout-content-full-width} var(--Layout-pane-width);
+      --Layout-template-areas: 'content pane';
+    }
+
+    // header divider
+
+    &.PageLayout--hasHeaderDivider {
+      .PageLayout-header {
+        // stylelint-disable-next-line primer/spacing
+        padding-bottom: max(var(--Layout-row-gap), var(--Layout-inner-spacing-min));
+        // stylelint-disable-next-line primer/borders
+        border-bottom: $border-width solid $Layout-divider-color;
+      }
+    }
+
+    // footer divider
+
+    &.PageLayout--hasFooterDivider {
+      .PageLayout-footer {
+        // stylelint-disable-next-line primer/spacing
+        padding-top: max(var(--Layout-row-gap), var(--Layout-inner-spacing-min));
+        // stylelint-disable-next-line primer/borders
+        border-top: $border-width solid $Layout-divider-color;
+      }
+    }
+
+    // pane divider
+
+    &.PageLayout--hasPaneDivider {
```

Comment:
`.PageLayout--hasPaneDivider` probably doesn't need to be a chained selector, but I can see why `.PageLayout--panePos-start` _does_ need to be chained.